Dockerfile

```
@@ -72,14 +73,19 @@ ENV NODE_ENV=production \
     NODE_OPTIONS="--max-old-space-size=4096"
 RUN bun --bun run build
 
-FROM builder AS release
+FROM base AS release
 
-COPY --from=dependencies /app/node_modules ./node_modules
+RUN adduser --disabled-password --gecos '' grimoire
```

```suggestion
RUN adduser --disabled-password --gecos '' --uid 10001 grimoire
```
We can be more specific here to avoid conflicts. What do you think?

---

Sure sounds good, I will also add it to the Alpine Image